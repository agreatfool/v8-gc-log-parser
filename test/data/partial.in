[28987:0x101804a00]    12910 ms: Mark-sweep 58.8 (95.9) -> 51.0 (96.9) MB, 3.8 / 0 ms (+ 8.2 ms in 101 steps since start of marking, biggest step 0.9 ms) [GC interrup

[51548:0x102004600]     3912 ms: pause=2.4 mutator=383.5 gc=s reduce_memory=

[51548:0x102004600]  

[51548:0x102004600]     3912 ms: pause=2.4 mutator=383.5 gc=s reduce_memory          

[51548:0x102004600]     3912 ms: pause=2.4 mutator=383.5 gc=s     

[15368:0x102004600] Memory allocator,   used: 375108 KB, available: 1091260 KB
[15368:0x102004600] New space,          used:   5428 KB, available:  10691 KB, committed:  32768 KB
[15368:0x102004600] Old space,          used: 201516 KB, available:  13911 KB, committed: 219092 KB
[15368:0x102004600] Code space,         used:   1287 KB, available:    974 KB, committed          
[15368:0x102004600] Map space,          used:    267 KB, available:    802 KB, committed:   1104 KB    
[15368:0x102004600] Large object space, used: 113283 KB, available: 1090219 KB, committed: 116000 KB   
[15368:0x102004600] All spaces,         used: 321784 KB, available: 1116598 KB, committed: 372036 KB
[15368:0x102004600] External memory reported:     16 KB  
[15368:0x102004600] Total time spent in GC  : 198.4 ms 
