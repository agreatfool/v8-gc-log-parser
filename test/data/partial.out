[
  {
    "pid": 28987,
    "isolate": "0x101804a00",
    "time_since_init": 12910,
    "data": {
      "collector": "mark_sweep",
      "start_object_size": 58.8,
      "end_object_size": 51,
      "start_memory_size": 95.9,
      "end_memory_size": 96.9,
      "pause": 3.8,
      "external_time": 0,
      "step_description": "+ 8.2 ms in 101 steps since start of marking, biggest step 0.9 ms"
    },
    "type": "trace"
  },
  {
    "pid": 51548,
    "isolate": "0x102004600",
    "time_since_init": 3912,
    "data": {
      "pause": 2.4,
      "mutator": 383.5,
      "gc": "s"
    },
    "type": "nvp"
  },
  {
    "pid": 51548,
    "isolate": "0x102004600",
    "time_since_init": 3912,
    "data": {
      "pause": 2.4,
      "mutator": 383.5,
      "gc": "s"
    },
    "type": "nvp"
  },
  {
    "pid": 51548,
    "isolate": "0x102004600",
    "time_since_init": 3912,
    "data": {
      "pause": 2.4,
      "mutator": 383.5,
      "gc": "s"
    },
    "type": "nvp"
  },
  {
    "pid": 15368,
    "isolate": "0x102004600",
    "type": "verbose",
    "data": {
      "memory_allocator": {
        "used": 375108,
        "available": 1091260
      },
      "new_space": {
        "used": 5428,
        "available": 10691,
        "committed": 32768
      },
      "old_space": {
        "used": 201516,
        "available": 13911,
        "committed": 219092
      },
      "code_space": {
        "used": 1287,
        "available": 974
      },
      "map_space": {
        "used": 267,
        "available": 802,
        "committed": 1104
      },
      "large_object_space": {
        "used": 113283,
        "available": 1090219,
        "committed": 116000
      },
      "all_spaces": {
        "used": 321784,
        "available": 1116598,
        "committed": 372036
      },
      "external_memory_reported": 16,
      "total_time_spent_in_gc": 198.4
    }
  }
]